src/node_modules/app/net.js

```diff
@@ -43,3 +43,35 @@ export async function request (
 		throw error;
 	}
 }
+
+export async function requestText (
```

Comment:
as this is almost identical to `request`, I'd add a `outputType` argument and use a single function with signature `request(fetch, method, url, outputType)`

Reply:
This actually belongs in the next PR, as it's only used by the compiler on the client side. I'll remove it from this PR